test/auto/injectorSpec.js

```diff
@@ -688,6 +688,24 @@ describe('injector', function() {
           expect(log.join('; ')).
             toBe('myDecoratedService:input,dependency1; myService:decInput; dec+origReturn');
         });
+
+
+        it('should allow for decorators to $inject', function() {
+          injector = createInjector(['ng', function($provide) {
+            $provide.value('dep', 'dependency');
```

Comment:
What's this used for ? Did yoy mean for the key to be `'key'` or something (to show it gets overwritten).